types/three/src/renderers/webgl/WebGLProgramParameters.d.ts

```diff
@@ -0,0 +1,199 @@
+import {
+    ColorSpace,
+    Combine,
+    DepthPackingStrategies,
+    GLSLVersion,
+    ShadowMapType,
+    ToneMapping,
+} from '../../constants.js';
+
+export interface WebGLProgramParameters {
```

Comment:
My preference is to avoid creating new declaration files that don't correspond to real JS files and to keep types as close to their source as possible. Can we move this interface to `WebGLPrograms`?